--- a/tofu/stalwart/main.tf
+++ b/tofu/stalwart/main.tf
@@ -242,10 +242,10 @@
     type              = "Manual"
   }
   dns_management = {
-    dns_server_id   = null
-    origin          = null
-    publish_records = null
-    type            = "Manual"
+    dns_server_id   = stalwart_dns_server_cloudflare.dkim.id
+    origin          = "manafishrov.com"
+    publish_records = ["dkim"]
+    type            = "Automatic"
   }
   is_enabled         = true
   logo               = null
